ArduCopter/GCS_Mavlink.cpp

```diff
@@ -750,16 +750,16 @@ GCS_MAVLINK_Copter::data_stream_send(void)
     }
 
     if (stream_trigger(STREAM_RAW_SENSORS)) {
-        send_message(MSG_RAW_IMU1);
-        send_message(MSG_RAW_IMU2);
-        send_message(MSG_RAW_IMU3);
+        send_message(MSG_RAW_IMU1);  // RAW_IMU, RAW_COMPASS
```

Comment:
Like I wrote in the other comment, no RAW_COMPASS, and SCALED_IMU2 should be here too.